tools/fuzz/fuzz.cpp

```diff
@@ -618,6 +618,34 @@ void DumpShader(spvtools::opt::IRContext* context, const char* filename) {
   DumpShader(binary, filename);
 }
 
+// Dumps |transformations| to file |filename| in binary format. Useful for
+// interactive debugging.
+void DumpTransformationsBinary(
```

Comment:
I might be wrong but it seems that these two functions are not used anywhere in the file.

Reply:
That's right - they're just useful when debugging; in the debugger you can set a breakpoint and then ask for various things to be dumped.